examples/stable/couchbase/blueprint-v2/couchbase-blueprint.yaml

```diff
@@ -20,7 +21,7 @@ actions:
           namespace: "{{ .Object.metadata.namespace }}"
       args:
         namespace: "{{ .Object.metadata.namespace }}"
-        image: ghcr.io/kanisterio/couchbase-tools:0.53.0
+        image: ghcr.io/kanisterio/couchbase-tools:0.57.0
```

Comment:
Wondering if release script is not updating the image tag

Reply:
since it replaces the last version with the next one, if there's ever an error, we'll miss future updates.

Reply:
I think we merged this after the release so the script missed these files.